docs/client_advanced.rst

```diff
@@ -171,17 +171,39 @@ Cookie Safety
 By default :class:`~aiohttp.ClientSession` uses strict version of
 :class:`aiohttp.CookieJar`. :rfc:`2109` explicitly forbids cookie
 accepting from URLs with IP address instead of DNS name
-(e.g. `http://127.0.0.1:80/cookie`).
+(e.g. ``http://127.0.0.1:80/cookie``).
 
 It's good but sometimes for testing we need to enable support for such
-cookies. It should be done by passing `unsafe=True` to
+cookies. It should be done by passing ``unsafe=True`` to
 :class:`aiohttp.CookieJar` constructor::
 
 
    jar = aiohttp.CookieJar(unsafe=True)
    session = aiohttp.ClientSession(cookie_jar=jar)
 
 
+.. _aiohttp-client-cookie-quoting-routine:
+
+Cookie Quoting Routine
+^^^^^^^^^^^^^^^^^^^^^^
+
+The client uses the :class:`~aiohttp.SimpleCookie` quoting routines
+conform to the RFC2109, which in turn references the character definitions
+from RFC2068. They provide a two-way quoting algorithm where any non-text
```

Comment:
There's a built-in Sphinx role for RFCs:
```suggestion
conform to the :rfc:`2109`, which in turn references the character definitions
from :rfc:`2068`. They provide a two-way quoting algorithm where any non-text
```

Reply:
Applied.